frontend/javascripts/oxalis/geometries/materials/edge_shader.ts

```diff
@@ -1,20 +1,25 @@
 import * as THREE from "three";
 import { COLOR_TEXTURE_WIDTH_FIXED } from "oxalis/geometries/materials/node_shader";
 import type { Uniforms } from "oxalis/geometries/materials/plane_material_factory";
+import { listenToStoreProperty } from "oxalis/model/helpers/listener_helpers";
+import shaderEditor from "oxalis/model/helpers/shader_editor";
+import { Store } from "oxalis/singletons";
+import _ from "lodash";
 
 class EdgeShader {
   material: THREE.RawShaderMaterial;
-  // @ts-expect-error ts-migrate(2564) FIXME: Property 'uniforms' has no initializer and is not ... Remove this comment to see the full error message
-  uniforms: Uniforms;
+  uniforms: Uniforms = {};
 
   constructor(treeColorTexture: THREE.DataTexture) {
     this.setupUniforms(treeColorTexture);
     this.material = new THREE.RawShaderMaterial({
-      // @ts-expect-error ts-migrate(2565) FIXME: Property 'uniforms' is used before being assigned.
       uniforms: this.uniforms,
       vertexShader: this.getVertexShader(),
       fragmentShader: this.getFragmentShader(),
+      transparent: true,
+      glslVersion: THREE.GLSL3,
```

Comment:
Are we requiring this somewhere else already or would this be a new requirement for systems running webknossos?

Reply:
The same flag is already passed in the node shader. Also, from my understanding, WebGL 2 support always implies support for GLSL 3.0. So, it's not a new requirement :)